lib/statsig/utils.ex

```diff
@@ -1,15 +1,17 @@
 defmodule Statsig.Utils do
-  def get_user_with_env(%{"statsigEnvironment" => env} = user) when not is_nil(env),
+  alias Statsig.User
+
+  def get_user_with_env(%User{statsigEnvironment: env} = user) when not is_nil(env),
     do: user
 
-  def get_user_with_env(%{} = user), do: put_env(user)
+  def get_user_with_env(%User{} = user), do: put_env(user)
 
-  def sanitize_user(user), do: Map.delete(user, "privateAttributes")
+  def sanitize_user(%User{} = user), do: %User{user | privateAttributes: nil}
 
-  defp put_env(user) do
+  defp put_env(%User{} = user) do
```

Comment:
Note: `put_env` has meaning in elixir, the `Application` module has a `put_env` function that adjusts the configuration.